Core/Abstractions/Repositories/ITaskRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using ddm = Domain.DataModels;
```

Comment:
`ddm` alias is not clear. Let's remove it and use explicit `Domain.DataModels.Task` behind in this file